recipes/default.rb

```diff
@@ -18,11 +18,15 @@
 
 pg = Chef::EncryptedDataBagItem.load(node['timesync']['databag'],
                                      'pg')
+secret_key = Chef::EncryptedDataBagItem.load(node['timesync']['databag'],
+				     'key')
 
 environment = {
   'PG_CONNECTION_STRING' => "postgres://#{pg['user']}:#{pg['pass']}@" \
     "#{pg['host']}:#{pg['port']}/#{pg['database_name']}",
-  'NODE_ENV' => 'production'
+  'NODE_ENV' => 'production',
+  'SECRET_KEY' => secret_key['key'],
+  'INSTANCE_NAME' => 'osuosl-timesync-staging'
```

Comment:
actually I would set it to something other than `osuosl-timesync-staging` as the default and we will set it to `osuosl-timesync-staging` in the role.